armory/data/datasets.py

```diff
@@ -965,6 +966,39 @@ def ucf101(
     )
 
 
+def ucf101_clean(
+    split: str = "train",
+    epochs: int = 1,
+    batch_size: int = 1,
+    dataset_dir: str = None,
+    preprocessing_fn: Callable = ucf101_canonical_preprocessing,
+    fit_preprocessing_fn: Callable = None,
+    cache_dataset: bool = True,
+    framework: str = "numpy",
+    shuffle_files: bool = True,
+) -> ArmoryDataGenerator:
+    """
+    UCF 101 Action Recognition Dataset with high quality MPEG extraction
+        https://www.crcv.ucf.edu/data/UCF101.php
+    """
+    preprocessing_fn = preprocessing_chain(preprocessing_fn, fit_preprocessing_fn)
+
+    return _generator_from_tfds(
+        "ucf101_clean/ucf101_1:2.0.0",
+        split=split,
+        batch_size=batch_size,
+        epochs=epochs,
+        dataset_dir=dataset_dir,
+        preprocessing_fn=preprocessing_fn,
+        as_supervised=False,
+        supervised_xy_keys=("video", "label"),
+        variable_length=bool(batch_size > 1),
+        cache_dataset=cache_dataset,
+        framework=framework,
+        shuffle_files=shuffle_files,
```

Comment:
With the (very) recently merged in PR #928, _generator_from_tfds now takes a context to facilitate the export process.

Reply:
Should be fixed now